test/functional/configuration/configuration.spec.js

```diff
@@ -252,29 +271,47 @@ test("returns full valid settings", async t => {
 });
 
 test("shows errors for empty required values", async t => {
-  await extensionViewController.init(t, {});
+  await extensionViewController.init(t, defaultInitInfo);
   await instances[0].nameField.clear(t);
+  await instances[0].imsOrgIdField.clear(t);
+  await instances[0].edgeDomainField.clear(t);
   await extensionViewController.expectIsNotValid(t);
   await instances[0].nameField.expectError(t);
   await instances[0].propertyIdField.expectError(t);
+  await instances[0].imsOrgIdField.expectError(t);
+  await instances[0].edgeDomainField.expectError(t);
+});
+
+test("shows errors for duplicate names", async t => {
+  await extensionViewController.init(t, defaultInitInfo);
+  await instances[0].propertyIdField.typeText(t, "PR123");
+  await addInstanceButton.click(t);
+  await instances[1].propertyIdField.typeText(t, "PR456");
+  // We'll expand the first instance before we validate to test that
+  // validation expands the invalid instance (in this case, the second one)
+  // Even though both accordion header labels are "alloy", this
+  // will select the first one.
+  await accordion.clickHeader(t, "ALLOY");
+  await extensionViewController.expectIsNotValid(t);
+  await instances[1].nameField.expectError(t);
```

Comment:
I just moved this test. It used to be below "shows errors for duplicate property IDs".